test/jasmine/tests/plot_api_react_test.js

```diff
@@ -1762,7 +1762,7 @@ describe('Plotly.react and uirevision attributes', function() {
         _run(fig, editEditable, checkAttrs(true), checkAttrs).then(done);
     });
 
-    it('@gl preserves editable: true name, colorbar title and parcoords constraint range via trace.uirevision', function(done) {
+    it('@noCI @gl preserves editable: true name, colorbar title and parcoords constraint range via trace.uirevision', function(done) {
```

Comment:
This one bothers me. Let's spend some time to try find out why it fails now.

Reply:
@archmoj I was able to make this test pass again in https://github.com/plotly/plotly.js/commit/f15c622698e4e8604301c804f9e69c92145005b9

Feel free to cherry-pick my commit onto your branch.

Reply:
@etpinard thanks very much for the help.
Done in https://github.com/plotly/plotly.js/pull/3966/commits/22283397911905e8b04dd05d97600bcc257265a7.